Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -144,6 +155,13 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
             collectionView.contentInset.bottom += typingIndicatorViewHeight
         }
         
+        view.addSubview(scrollToLatestMessageButton)
+        collectionView.bottomAnchor.pin(equalToSystemSpacingBelow: scrollToLatestMessageButton.bottomAnchor).isActive = true
```

Comment:
nit: Can we do it vice versa? 😄 i.e. pin the bottom to the collection view? At first glance it feels like a constraint for another random view is added

Reply:
I hate it but not without hardcoding a value :( 
The button needs to be below collection view, If I would swap them, the system spacing would be in the opposite direction

Reply:
if it really bothers you, you can add `pin(equalToSystemSpacingAbove:)` and just flip the views inside. Let's please do it later during the audit of this VC.